### PR TITLE
alpaca: remove method [wip]

### PR DESCRIPTION
**Pull request by sc0Vu**

### CODE REVIEW

File: ts/src/alpaca.ts [hunk 1]
```diff
@@ -444,8 +441,7 @@ export default class alpaca extends Exchange {
         if (since !== undefined) {
             request['start'] = this.yyyymmdd (since);
         }
-        const method = this.safeString (this.options, 'fetchOHLCVMethod', 'cryptoPublicGetCryptoBars');
-        const response = await this[method] (this.extend (request, params));
+        const response = await this.cryptoPublicGetCryptoBars (this.extend (request, params));
```
Comment (carlosmiei) [inline ts/src/alpaca.ts:444]: @sc0Vu we have to keep the ability of using `cryptoPublicGetCryptoLatestBars` as well

File: ts/src/alpaca.ts [hunk 2]
```diff
@@ -337,8 +335,7 @@ export default class alpaca extends Exchange {
         if (limit !== undefined) {
             request['limit'] = limit;
         }
-        const method = this.safeString (this.options, 'fetchTradesMethod', 'cryptoPublicGetCryptoTrades');
-        const response = await this[method] (this.extend (request, params));
+        const response = await this.cryptoPublicGetCryptoTrades (this.extend (request, params));
```
Comment (carlosmiei) [inline ts/src/alpaca.ts:338]: same here

File: ts/src/alpaca.ts (hunk 1 above)
Comment (sc0Vu) [reply ts/src/alpaca.ts:444]: @carlosmiei it seems `cryptoPublicGetCryptoLatestTrades` and `cryptoPublicGetCryptoLatestBars` were removed, and I can't find in their documentation neither.

Comment (carlosmiei) [reply ts/src/alpaca.ts:444]: @sc0Vu ok I will check it

File: ts/src/alpaca.ts (hunk 2 above)
Comment (carlosmiei) [reply ts/src/alpaca.ts:338]: @sc0Vu https://alpaca.markets/docs/crypto/crypto-data/
image
image

maybe we need to add them to the implicit API?

Comment (sc0Vu) [reply ts/src/alpaca.ts:338]: @carlosmiei uh, they have upgraded to beta3, and added locale parameter (required) in path, eg `cryptoPublicGetCryptoLocLatestBars ()`, we probably need to upgrade as well.

https://docs.alpaca.markets/reference/cryptolatestbars

https://docs.alpaca.markets/reference/cryptolatesttrades

**Pull request closed without merging**